signalfx-java/src/main/java/com/signalfx/signalflow/ChannelMessage.java

```diff
@@ -404,11 +404,15 @@ public long getTimestampMs() {
     }
 
     /**
-     * Message received when the computation encounters errors during its initialization.
+     * Message received when the computation encounters errors during its initialization. Because the error that is
+     * returned might have one of two sets of contents, this has both an `errors` and a `message` accessor. We'll
+     * be liberal here so the code can decide how to show this error later. Optimally, these should be different
+     * message types, but that's a lot work work.
      */
     public static class ErrorMessage extends ChannelMessage {
 
         protected ArrayList<Object> errors;
+        protected String message;
```

Comment:
I think you should also capture and expose `errorCode` so you can properly differentiate between 4xx and 5xx classes of errors.